src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+        Random random = new Random();
+        Ball a = new Ball();
+        a.setNumber(random.nextInt(100));
+        a.setColor(ColorSupplier.getRandomColor());

```

Comment:
Random and ColorSupplier should be a class field

Reply:
not fixed